grammar/TypeQL.g4

```diff
@@ -267,20 +306,21 @@ TRUE            : 'true'        ;
 FALSE           : 'false'       ;
 STRING_         : '"'  (~["\\] | ESCAPE_SEQ_ )* '"'
                 | '\'' (~['\\] | ESCAPE_SEQ_ )* '\''    ;
-LONG_           : ('+' | '-')? [0-9]+                   ;
```

Comment:
@flyingsilverfin , I had to remove the unary operators from the token because `?x = 5 +2` would not parse properly.